dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -93,13 +94,13 @@
     <Import Project="..\Scroller\Scroller.vcxitems" Label="Shared" />
     <Import Project="..\ScrollViewer\ScrollViewer.vcxitems" Label="Shared" Condition="$(BuildLeanMuxForTheStoreApp) != 'true'" />
     <Import Project="..\Lights\Lights.vcxitems" Label="Shared" />
-    <Import Project="..\Repeater\Repeater.vcxitems" Label="Shared" />
+    <Import Project="..\Repeater\Repeater.vcxitems" Label="Shared" Condition="$(RepeaterExcluded) != 'true'"/>
     <Import Project="..\SwipeControl\SwipeControl.vcxitems" Label="Shared" Condition="$(BuildLeanMuxForTheStoreApp) != 'true'" />
     <Import Project="..\TwoPaneView\TwoPaneView.vcxitems" Label="Shared" Condition="$(BuildLeanMuxForTheStoreApp) != 'true'" />
     <Import Project="..\TestHooks\TestHooks.vcxitems" Label="Shared" />
     <Import Project="..\PullToRefresh\PTRTracing\PTRTracing.vcxitems" Label="Shared" Condition="$(BuildLeanMuxForTheStoreApp) != 'true'" />
     <Import Project="..\SplitButton\SplitButton.vcxitems" Label="Shared" Condition="$(BuildLeanMuxForTheStoreApp) != 'true'" />
-    <Import Project="..\LayoutPanel\LayoutPanel.vcxitems" Label="Shared" />
+    <Import Project="..\LayoutPanel\LayoutPanel.vcxitems" Label="Shared" Condition="$(LayoutPanelExcluded) != 'true'"/>
```

Comment:
We should probably delete all references to BuildLeanMuxForTheStoreApp since this is a more general version of that. If we ever needed to create a similar "lean mux", it would probably take the form of a custom version of FeatureAreaFilterForInnerLoop.props.